content/appendices/templates.md

```diff
@@ -53,7 +53,7 @@ The package meets the readme requirements below:
 The README should include, from top to bottom:
 
 - [ ] The package name
-- [ ] Badges for continuous integration and test coverage, the badge for pyOpenSci peer-review once it has started (see below), a repostatus.org badge, and any other badges. If the README has many more badges, you might want to consider using a table for badges, see this example, that one and that one. Such a table should be more wide than high.
+- [ ] Badges for continuous integration and test coverage, a repostatus.org badge, and any other badges. If the README has many more badges, you might want to consider using a table for badges ([see this example](https://github.com/ropensci/drake)). Such a table should be more wide than high. (Note that the badge for pyOpenSci peer-review will be provided upon acceptance.) 
```

Comment:
@jlpalomino this url does look weird to me. 

([see this example](https://github.com/ropensci/drake))

Reply:
i wonder if it doesnt' like the `()` around the link?? i succest you put the () around the text within the link like this: [(see this example)] but i'm not sure if that will work or be weird either.

Reply:
@lwasser I was trying to include the link within a parenthesis, rather than as part of the sentence, but I will make that change now and see if it fixes the problem.

Reply:
I removed the () altogether, but the same error comes up.